test/reference.jl

```diff
@@ -153,7 +156,11 @@ end
                 dispatch=true,
                 dispatch_delay=20,
             ),
-            Tests(; project=true),
+            Tests(;
+                project=true,
+                aqua=true,
```

Comment:
```suggestion
                aqua_version=v"0.6.5",
```
to fix it so that the manifest doesn't change!